### PR TITLE
Limit impala to single GPU training

### PR DESCRIPTION
**Pull request by zenghsh3**

### CODE REVIEW

File: examples/IMPALA/train.py [hunk 1]
```diff
@@ -54,6 +55,10 @@ def __init__(self, config):
         self.agent = AtariAgent(algorithm, obs_shape, act_dim,
                                 self.learn_data_provider)
 
+        if machine_info.is_gpu_available():
+            assert get_gpu_count() == 1, 'Only support training in single GPU,\
+                    Please set environment variable: `export CUDA_VISIBLE_DEVICES=[GPU_ID_YOU_WANT_TO_USE]` .'
```
Comment (yobobobo) [inline examples/IMPALA/train.py]: `export CUDA_VISIBLE_DEVICES=[GPU_ID_TO_USE]`

Comment (zenghsh3) [reply examples/IMPALA/train.py]: done

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
LGTM.

**Comment by TomorrowIsAnOtherDay** [comment]
resolved #151

**Pull request merged**